Formula/dump1090-mutability.rb

```diff
@@ -0,0 +1,49 @@
+class Dump1090Mutability < Formula
+  desc "ADS-B Ground Station System for RTL-SDR"
+  homepage "https://packages.ubuntu.com/jammy/dump1090-mutability"
+  url "http://archive.ubuntu.com/ubuntu/pool/universe/d/dump1090-mutability/dump1090-mutability_1.15~20180310.4a16df3+dfsg.orig.tar.gz"
```

Comment:
Is this the latest one?

Reply:
I think this is the latest version *released* (The Formula Cookbook requires a stable, tagged version.). On dump1090-mutability's [git repo](https://github.com/mutability/dump1090/), there are a few commits after 4a16df3, but these commits are not released or tagged.